src/replace/module.coffee

```diff
@@ -10,10 +16,24 @@ module.exports = (path, stub) ->
   realThing = require(quibble.absolutify(path))
   fakeThing = imitate(realThing, path)
   quibble(path, wrapIfNeeded(fakeThing, realThing))
-  return fakeThing
+  return _.merge(fakeThing, imitateProperties(realThing))
 
 wrapIfNeeded = (fakeThing, realThing) ->
   if isConstructor(realThing)
     wrapWithConstructor(fakeThing)
   else
     fakeThing
+
+imitateProperties = (realThing) ->
+  return {} unless _.isFunction(realThing)
+  _.reduce imitatableKeys(realThing), wrapKey(realThing), {}
+
+imitatableKeys = (realThing) ->
+  _.keys(realThing).filter (key) ->
```

Comment:
This ignores inherited properties. Not sure if this should be the desired functionality.